simphony/io/tests/test_h5_lattice.py

```diff
@@ -73,9 +108,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
-        return H5Lattice.create_new(
-            self.handle.root, 'foo', type_, base_vect,
-            size, origin, record=CustomRecord)
+        self.group = self.handle.create_group(
```

Comment:
hmm..., it looks that the test is a little problematic. In theory the H5Lattice should not care about the name that the parent group has, but this means that we need to revisit the test template which I think is out of the scope of this PR. lets keep the current implementation and open an issue to revisit the tests (there is always space for cleanup).